src/main/java/com/netchill/services/streaming/StreamingService.java

```diff
@@ -0,0 +1,44 @@
+package com.netchill.services.streaming;
+
+import com.netchill.api.moviedb.MovieDbApiClient;
+import com.netchill.api.moviedb.models.Trailer;
+import com.netchill.api.moviedb.models.TrailerKey;
+import com.netchill.db.dao.movie.MovieDao;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+
+@Singleton
+public class StreamingService {
+    private MovieDbApiClient movieDbApiClient;
+
+    @Inject
+    private StreamingService(MovieDbApiClient movieDbApiClient, MovieDao movieDao){
+        this.movieDbApiClient = movieDbApiClient;
+    }
+
+    public TrailerKey getSerieTrailerById(Long id) {
+        Trailer trailer = movieDbApiClient.getSerieTrailerById(id);
+        List<TrailerKey> trailerKeys = trailer.getResults();
+        if(trailerKeys.isEmpty()) return new TrailerKey();
+        for(int i=0; i < trailerKeys.size(); i++){
+            if (trailerKeys.get(i).getType().equals("Trailer")) {
+                return trailerKeys.get(i);
+            }
+        }
```

Comment:
structure ton code en étapes via des sauts de ligne, afin de le rendre plus facile à comprendre:

1. Récupération des données 
2. vérification des données
3. traitement des données

```suggestion
        Trailer trailer = movieDbApiClient.getSerieTrailerById(id);
        List<TrailerKey> trailerKeys = trailer.getResults();
        
        if(trailerKeys.isEmpty()) {
            return new TrailerKey();
        }
        
        for(int i=0; i < trailerKeys.size(); i++){
            if (trailerKeys.get(i).getType().equals("Trailer")) {
                return trailerKeys.get(i);
            }
        }
```